src/test/resources/org/rust/lang/core/parser/fixtures/ill-formed/shifts.txt

```diff
@@ -0,0 +1,99 @@
+FILE
+  RustFileModItemImpl(FILE_MOD_ITEM)
+    RustFnItemImpl(FN_ITEM)
+      PsiElement(fn)('fn')
+      PsiWhiteSpace(' ')
+      PsiElement(<IDENTIFIER>)('expressions')
+      RustGenericParamsImpl(GENERIC_PARAMS)
+        <empty list>
+      RustFnParamsImpl(FN_PARAMS)
+        PsiElement(()('(')
+        PsiElement())(')')
+      PsiWhiteSpace(' ')
+      RustBlockImpl(BLOCK)
+        PsiElement({)('{')
+        PsiWhiteSpace('\n    ')
+        RustLitExprImpl(LIT_EXPR)
+          RustNumericLiteralImpl(<INTEGER>)('1')
+        PsiErrorElement:';' or '}' expected, got '>'
+          <empty list>
+  PsiWhiteSpace(' ')
+  PsiElement(>)('>')
+  PsiElement(DUMMY_BLOCK)
```

Comment:
I don't like the mess which starts here, but I don't know how to make recovering properly here (some tricks in `collapse` method?)